google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/SubscriberClientSnippets.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
repo checkstyle requires ARR.('^ \* Copyright \d\d\d\d Google Inc\. All Rights Reserved\.$'. [RegexpHeader])
 Leaving it in for now : will resolve it separately for google-cloud-examples.